charts/shoot-cloud-config/charts/original/templates/kubelet/_kubelet.flags

```diff
@@ -1,67 +1,44 @@
 {{- define "kubelet-flags" -}}
-{{- if semverCompare "< 1.10" .kubernetes.version -}}
```

Comment:
I've pretty much removed all version specific configurations except for

1. flags added / removed in different versions.
1. flags being set based on some specific value.